mobile/library/kotlin/io/envoyproxy/envoymobile/BUILD

```diff
@@ -31,8 +31,8 @@ android_artifacts(
     archive_name = "envoy_xds",
```

Comment:
We also need to update https://github.com/envoyproxy/envoy/blob/ff1b8e622474ad4dcfe19a9539482df42e6d8b20/mobile/library/kotlin/io/envoyproxy/envoymobile/BUILD#L17

Reply:
good catch, done